dashboard/app/controllers/api/v1/assessments_controller.rb

```diff
@@ -1,8 +1,14 @@
 class Api::V1::AssessmentsController < Api::V1::JsonApiController
   include LevelsHelper
+
+  before_filter :load_from_cache
   load_and_authorize_resource :section
```

Comment:
It looks like `@section` doesn't get used in the `:index` route.  We should probably prevent its `load_and_authorize_resource` call from running for that method.